frontend/src/components/MeetingRoom/ParticipantListButton/ParticipantListButton.tsx

```diff
@@ -43,12 +45,14 @@ const ParticipantListButton = ({
           overlap="circular"
         >
           <ToolbarButton
+            data-testid="participant-list-toggle"
```

Comment:
Not a toggle!

Reply:
I'm starting to suspect you don't like toggles 😂

Reply:
It might be excessive, but I'm thinking of adding a lint rule for this 😂